preview_generator/preview/builder/image__wand.py

```diff
@@ -113,7 +113,7 @@ def build_jpeg_preview(
                     buffer = result.read(1024)
 
     def image_to_jpeg_wand(
-        self, jpeg: typing.Union[str, typing.IO[bytes]], preview_dims: ImgDims = None
```

Comment:
Why is preview_dims default value remove here ?

Reply:
Because this value, if used, were causing an exception a few lines after:

```
preview_dims = ImgDims(width=preview_dims.width, height=preview_dims.height)
```

So I bet it was in fact not optional.